hack/recurl.sh

```diff
@@ -4,10 +4,12 @@ set -eux -o pipefail
 file=$1
 url=$2
 
-# loop forever
-while ! curl -L -o "$file" -- "$url" ;do
-  echo "sleeping before trying again"
-  sleep 10s
-done
+if [ ! -f "$file" ]; then
```

Comment:
I had an issue where this fell into an endless loop of failing to write a file that was already there.